sdk-core/src/test/java/care/data4life/sdk/wrapper/URLEncodingTest.kt

```diff
@@ -20,7 +20,7 @@ import org.junit.Test
 import kotlin.test.assertEquals
 import kotlin.test.assertTrue
 
-class UrlEncodingTest {
+class URLEncodingTest {
```

Comment:
This should be lowercase, abreviations are also camel cased. This is easier to read.

Reply:
Done

Reply:
done